webcompat/helpers.py

```diff
@@ -722,3 +722,15 @@ def ab_init(response):
             response.set_cookie(exp_id, var, max_age=max_age)
 
     return response
+
+
+def get_extra_labels(form):
+    """If extra_labels exists in current session, use it,
+    otherwise use the value coming from form."""
```

Comment:
nit: The first line is a short summary of the function.
Then  two returns and the explanation if necessary.

I would probably do something along.

```python
"""Extract extra_labels.

If extra_labels exists in current session, use it,
otherwise use the value coming from form.
"""